blackbox/docs/todo.txt

```diff
@@ -0,0 +1,354 @@
+admin/optimization.rst:.. highlight:: psql
```

Comment:
was this intentional checked in?

Reply:
I will have to look at this together with @mechanomi.